crates/nu-cli/src/eval_file.rs

```diff
@@ -78,8 +81,7 @@ pub fn evaluate_file(
             );
             std::process::exit(1);
         });
-
-    engine_state.start_in_file(Some(file_path_str));
+    engine_state.script = Some(file_path.clone());
```

Comment:
Personally I would prefer this to remain an accessor instead of assignment to `pub` fields, if we intend to do more logic when switching the file.

Reply:
I removed `start_in_file()` because its usage is inconsistent (only two occurances throughout the entire codebase). We can re-introduce an abstraction later if necessary.